StortSpelProjekt/Engine/src/Renderer/QuadManager.cpp

```diff
@@ -0,0 +1,166 @@
+#include "stdafx.h"
+#include <vector>
+#include "QuadManager.h"
+
+#include "../Renderer/Texture/Texture.h"
+#include "../Renderer/DescriptorHeap.h"
+#include "../Renderer/GPUMemory/Resource.h"
+#include "../Renderer/GPUMemory/ShaderResourceView.h"
+#include "../Renderer/Mesh.h"
+#include "../Renderer/Renderer.h"
+
+#include "../Misc/Window.h"
+#include "../Misc/AssetLoader.h"
+
+#include "../Events/EventBus.h"
+
+QuadManager::QuadManager()
+{
+}
+
+QuadManager::~QuadManager()
+{
+	EventBus::GetInstance().Unsubscribe(this, &QuadManager::pressed);
+
+	delete m_pQuad;
```

Comment:
This is "unsafe", you don't have a predefined behavior for initing the m_pQuad. If CreadQuad is never called, this will cause a crash.

Reply:
true, you have a good eye